packages/@tinacms/graphql/src/mdx/stringify.ts

```diff
@@ -439,6 +439,64 @@ ${out}
       }
 
     case 'text':
+      if (node?.bold && node?.italic) {
+        return {
+          type: 'strong',
+          children: [
+            {
+              type: 'emphasis',
+              children: [
+                {
+                  type: 'text',
+                  value: node.text || '',
+                },
+              ],
+            },
+          ],
+        }
+      }
+
+      if (node?.bold) {
+        return {
+          type: 'strong',
+          children: [
+            {
+              type: 'text',
+              value: node.text || '',
+            },
+          ],
+        }
+      }
+
+      if (node?.italic) {
+        return {
+          type: 'emphasis',
+          children: [
+            {
+              type: 'text',
+              value: node.text || '',
+            },
+          ],
+        }
+      }
+
+      // type delete is only supported via an extension
```

Comment:
Interesting comments, not sure what to do here, thoughts @jeffsee55 ?

Reply:
Yeah I think we should just remove strikethrough for now, it can be implemented with a custom mdx component if needed.